added RVM Description to SMP Documentation (#108)
layer5io/service-mesh-performance: CONTRIBUTING.md

<a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>

### Set up your development environment

>  Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/) and also install Ruby Version Manager [RVM](https://rvm.io) - a command-line tool which allows you to easily install, manage and work with multiple ruby environments on your local machine. Alternatively, if you're running Windows 10, update to version 1903 Build 18362 or higher. You can also shift to Windows Subsystem for Linux [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10)

Reviewer: ```suggestion
 Note: Windows users can run Jekyll by following the [Windows Installation Guide,](https://jekyllrb.com/docs/installation/windows/) for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
 Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:

-  [WSL1](https://docs.microsoft.com/en-us/windows/wsl/install-win10)  for Windows build version 1607 or higher.
-  [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#update-to-wsl-2) for Windows build version 1903 or higher.

After successfully installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```

Reviewer: @oyinmi Good work! A few changes to make here.

Author: @chandrashritii Thank you. Alright.

Author: @chandrashritii I've made the correction, thank you.

<a href="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" ><img src="https://user-images.githubusercontent.com/7570704/64490167-98906400-d25a-11e9-8b8a-5f465b854d49.png" width="50%"><a>

### Set up your development environment

> Note: Windows users can run Jekyll by following the [Windows Installation Guide,](https://jekyllrb.com/docs/installation/windows/) for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 

Reviewer 2: ```suggestion
Note: Windows users can run Jekyll by following the [Windows Installation Guide](https://jekyllrb.com/docs/installation/windows/), for Jekyll. This includes installing the Ruby Version Manager [RVM](https://rvm.io) - a command-line tool that allows you to easily install, manage, and work with multiple ruby environments on your local machine. 
```

Author: Thank you @JubayerJoy

>  Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:

Reviewer: ```suggestion
 Alternatively, if you are running Windows 10, you may install the Windows Subsystem for Linux:
 
```

-  [WSL1](https://docs.microsoft.com/en-us/windows/wsl/install-win10)  for Windows build version 1607 or higher.
-  [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10#update-to-wsl-2) for Windows build version 1903 or higher.

> After successfully installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.

Reviewer: ```suggestion

After successfully installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```

Author: Alright,  will fix it. Thank you

Reviewer 2: ```suggestion
After successful installation, proceed with the following [steps](https://jekyllrb.com/docs/installation/windows/#installation-via-bash-on-windows-10) to run Jekyll on WSL.
```
